LockwiseXCUITests/LockwiseXCUITests.swift

```diff
@@ -160,7 +160,7 @@ class LockwiseXCUITests: BaseTestCase {
         if  iPad() {
             XCTAssertEqual(aMatches, 111)
```

Comment:
We need to update also the number of logins shown on iPad, it should be 110 now..